probe-rs/src/flashing/flasher.rs

```diff
@@ -48,17 +49,20 @@ impl Operation for Verify {
 /// A structure to control the flash of an attached microchip.
 ///
 /// Once constructed it can be used to program date to the flash.
-pub(super) struct Flasher<'session> {
+pub(super) struct Flasher<'session, 'progress> {
     session: &'session mut Session,
     core_index: usize,
     flash_algorithm: FlashAlgorithm,
+    #[allow(dead_code)]
```

Comment:
It is not used if the"rtt" feature is not set. But it would make for insanely ugly code if I featuregated the lifetime for the progress handle. So I always have this handle and allow dead code.

Reply:
Just tried removing it, and didn't get a warning, even without the `rtt` feature.